test/parallel/test-stream2-readable-wrap-error.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+const Readable = require('_stream_readable');
+const EE = require('events').EventEmitter;
+
+const oldStream = new EE();
+oldStream.pause = () => {};
+oldStream.resume = () => {};
+
+{
+  const r = new Readable({ autoDestroy: true })
+    .wrap(oldStream)
+    .on('error', common.mustCall(() => {
+      assert.strictEqual(r._readableState.errorEmitted, true);
+      assert.strictEqual(r._readableState.errored, true);
+      assert.strictEqual(r.destroyed, true);
+    }));
+  oldStream.emit('error', new Error());
+}
+
+{
+  const r = new Readable({ autoDestroy: false })
+    .wrap(oldStream)
+    .on('error', common.mustCall(() => {
+      assert.strictEqual(r._readableState.errorEmitted, true);
+      assert.strictEqual(r._readableState.errored, true);
```

Comment:
This should not be part of backport

Reply:
I think the above line should be removed as well, as per

```
AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:
+ actual - expected

+ undefined
- true
    at Readable.<anonymous> (/home/runner/work/node/node/test/parallel/test-stream2-readable-wrap-error.js:27:14)
    at Readable.<anonymous> (/home/runner/work/node/node/test/common/index.js:361:15)
    at Readable.emit (events.js:314:20)
    at errorOrDestroy (internal/streams/destroy.js:108:12)
    at EventEmitter.<anonymous> (_stream_readable.js:1058:5)
    at EventEmitter.emit (events.js:326:22)
    at Object.<anonymous> (/home/runner/work/node/node/test/parallel/test-stream2-readable-wrap-error.js:31:13)
    at Module._compile (internal/modules/cjs/loader.js:1015:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)
    at Module.load (internal/modules/cjs/loader.js:879:32) {
  generatedMessage: true,
  code: 'ERR_ASSERTION',
  actual: undefined,
  expected: true,
  operator: 'strictEqual'
}
```